### DISCUSSION TITLE
How to append to delta table using Rust?

### DISCUSSION THREAD

**Discussion by finlaydotb** [delta-io/delta-rs#1567] [Q&A]
Hi all, I Just asked a question on StackOverflow on how to perform append write using delta-rs. Similar code in Python works, but the Rust version panics with ProtocolChanged  and I am not sure why.

The question is here https://stackoverflow.com/questions/76779118/how-to-append-to-delta-table-using-rust

If anyone knows how to fix this, I will appreciate an answer to the question on StackOverflow

**Comment by rtyler** [comment]
@finlaydotb The linked page returns a 404, is this still a problem you're facing?
